partner_validate_email/requirements.txt

```diff
@@ -1,2 +1,2 @@
 validate-email==1.3
-pydns==2.3.6
\ No newline at end of file
+py3dns==3.2.1
```

Comment:
Ok, I realize it's here.
Can you move it to the root of the repository?

We should have only one `requirements.txt` file at the root, including all the pip requirements from all the modules on this repo

Reply:
Don't forget to move this to the root of the repository